ocaml/xapi/cli_frontend.ml

```diff
@@ -2611,7 +2611,7 @@ add a mapping of 'path' -> '/tmp', the command line should contain the argument
 
 		"lvhd-enable-thin-provisioning",
 		{
-			reqd=["sr-uuid"; "initial-allocation"; "allocation-quantum"];
+			reqd=["host-uuid"; "sr-uuid"; "initial-allocation"; "allocation-quantum"];
```

Comment:
Is it worth handling the standard host-selectors (including name) rather than insisting on the uuid?